backend/account/tests/test_business.py

```diff
@@ -1,14 +1,14 @@
 import datetime
 from django.test import TestCase
-from django.core.exceptions import ObjectDoesNotExist
-from account import business, query
+from account import business
+from column.business import get_columns
 
 
 class CreateUserTests(TestCase):
     def test_create_user(self):
```

Comment:
This test is still focusing on columns and could be migrated to that package under the name `test_default_columns`. Can we test for User creation success with other parameters that do not depend on columns?

Reply:
Unit tests will be revised in #107 since the majority of our tests currently are dependant on other elements. Not worried about moving this test before we correct tests to be isolated.